Sources/TorusUtils/Helpers/Metadata.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+import BigInt
+import CryptorECC
+
+func convertMetadataToNonce(params: [String: Any]?) -> BigUInt {
+    guard let params = params, let message = params["message"] as? String else {
+        return BigUInt(0)
+    }
+    return BigUInt(message, radix: 16) ?? BigUInt(0)
```

Comment:
you already catched BigUInt(0) in line 7, so I think this is unneeded
```suggestion
    return BigUInt(message, radix: 16)
```

Reply:
This is for in case message is available but have not parsed correctly. Instead of returning nil, returning BigUInt(0)

Reply:
In typescript version, it isn't returning 0 when not parsed correctly,
`  return new BN(params.message, 16);
`
I think we should follow their intention.

Reply:
return new BN(params.message, 16)!